<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlotLimitsDlg</class>
 <widget class="QDialog" name="PlotLimitsDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>262</width>
    <height>270</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Plot Limits</string>
  </property>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>192</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Scans To Plot :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="nscansPlotLE">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>192</y>
     <width>125</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="DNMinLE">
   <property name="geometry">
    <rect>
     <x>119</x>
     <y>66</y>
     <width>125</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>24</y>
     <width>82</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>DN Maximum</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="PPMMMinLE">
   <property name="geometry">
    <rect>
     <x>119</x>
     <y>150</y>
     <width>125</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="PPMMMaxLE">
   <property name="geometry">
    <rect>
     <x>119</x>
     <y>108</y>
     <width>125</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>150</y>
     <width>99</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>PPMM Minimum </string>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>232</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>66</y>
     <width>78</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>DN Minimum</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>108</y>
     <width>102</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>PPMM Maximum </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="DNMaxLE">
   <property name="geometry">
    <rect>
     <x>119</x>
     <y>24</y>
     <width>125</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>PlotLimitsDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>PlotLimitsDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
